Formula/k3d.rb

```diff
@@ -42,7 +46,7 @@ def install
   end
 
   test do
-    assert_match "k3d version v#{version}\nk3s version latest (default)", shell_output("#{bin}/k3d --version")
+    assert_match "k3d version v#{version}", shell_output("#{bin}/k3d version | head -1")
```

Comment:
```suggestion
    assert_match "k3d version v#{version}", shell_output("#{bin}/k3d version")
```
`assert_match` only matches substrings, so you don't need to pipe to `head`. If you really only want the first line, you can use `.split.first`. (`split` might needs to take a `"\n"` argument; I forget if this splits new lines by default.)

Reply:
Good to know, thanks!

Reply:
:heavy_check_mark: